src/components/ViewFieldIngelogdExtraInfo/ViewFieldIngelogdExtraInfo.js

```diff
@@ -19,6 +14,15 @@ const eigenarenProperties = [
     'Opdrachtgever',
 ]
 
+/**
+ * Component that renders the ViewFieldIngelogdExtraInfo component and displays a owner list and a button to open the list in the admin area.
+ *
+ * @component
+ *
+ * @param {object} crudObject - Parameter containing a collection of data from the parent state.
```

Comment:
This should be more specific, what is the context of this collection of data from the parent state? I would propose:
crudObject - Contains the object we want to display the extra information of